### ISSUE TITLE
Feature: epub download

### ISSUE THREAD

**Issue by domschl** [arxiv-vanity/arxiv-vanity#78]
Since you've done already the hard work to convert, it would be nice to be able to download a paper as epub.

**Comment by bfirsh** [comment]
Thanks for the suggestion! This is actually something plain Pandoc does pretty well: https://pandoc.org/epub.html

**Comment by domschl** [comment]
Yes, agreed. I was thinking along the lines of an epub-download-button for either lazy or mobile users...

**Comment by bfirsh** [comment]
A +1 here! https://twitter.com/flowing/status/923109563758456832

**Comment by JiamingSuen** [comment]
+1 for this feature! Reading arxiv paper on kindle would be really nice! Maybe adding a "Send to Kindle" button later.

**Comment by devmessias** [comment]
@JiamingSuen You can use the official extension "send to kindle" provided by Amazon. 
[https://chrome.google.com/webstore/detail/send-to-kindle-for-google/cgdjpilhipecahhcilnafpblkieebhea?hl=en](https://chrome.google.com/webstore/detail/send-to-kindle-for-google/cgdjpilhipecahhcilnafpblkieebhea?hl=en)

**Comment by jkukul** [comment]
To all the people interested in reading arXiv papers on Kindle - when you send the original pdf to your Kindle email address and **type 'convert' in the subject line**, the pdf will be converted to the Kindle format (`.azw`). This can be an alternative e.g. when you're on mobile and the browser extension is not available. 

The result is pretty good and gives you all the benefits of an ebook format. Unfortunately, things like tables and equations are mangled, but all the other alternatives suggested in this thread (Pandoc's `html -> epub` or Amazon's browser extension) suffer the same problem.

**Comment by bfirsh** [comment]
We quite often get requests about this.

My comment above is now out of date. We now use LaTeXML, not Pandoc. But LaTeXML also supports ePub output!

There are a few parts to this work I think:

1) Playing around with ePub output from LaTeXML and seeing what it's like. It might need transformation, like Engrafo does to HTML, in which case perhaps Engrafo shoudl have an ePub output, or perhaps something else entirely is needed.
2) Render in arXiv vanity somehow. My feeling is that ePubs should be generated on-demand separately from the HTML, because much fewer people will be reading them. There needs to be some logic that'll generate ePub renders for papers and cache them, like is done with HTML.
3) User interface. This is downloading something rather than just displaying it, and it needs to sit there and render for a bit, so some consideration to how the UI is going to work is needed.

**Comment by bquast** [comment]
+1 for this, and thanks for already making it relatively easy!

**Comment by turian** [comment]
@bfirsh Is ePub availble now? There ae a few pares  I want to read on my kndle

**Comment by bquast** [comment]
Still also hoping for this
